api/v1.inc

```diff
@@ -25,6 +25,9 @@ $router->add_route("GET", "v1/projects/genres", "api_v1_projects_genres");
 $router->add_route("GET", "v1/projects/languages", "api_v1_projects_languages");
 $router->add_route("GET", "v1/projects/states", "api_v1_projects_states");
 $router->add_route("GET", "v1/projects/pagerounds", "api_v1_projects_pagerounds");
+$router->add_route("GET", "v1/projects/charsuites", "api_v1_projects_charsuites");
```

Comment:
I know our project ids cannot resemble words like "states", or "charsuites", but for routing purposes, does order matter? That way an ambiguous route like "v1/projects/states" that can match both won't ever hit the "v1/projects/:projectid" route with :projectid set to "states"? I guess if it is working, it doesn't matter.

Reply:
The ApiRouter evaluates static routes over parameter-based routes so the order isn't important. Although to be fair I had to look back at the code to confirm that for certain.